torntools/scripts/content/properties/ttProperties.js

```diff
@@ -1,16 +1,18 @@
 requireDatabase().then(() => {
 	console.log("TT - Properties");
 	let observer = new MutationObserver(addPropertyValues);
-	observer.observe(doc.find("div#properties-page-wrap"), {childList: true});
+	observer.observe(doc.find("div#properties-page-wrap"), { childList: true });
 	addPropertyValues();
 });
 
 function addPropertyValues() {
-	let propertyValue = null;
 	for (let property of doc.find(".properties-list").children) {
+		let propertyValue = null;

```

Comment:
I think what DKK meant was to remove this completely and use the `let` where you actually create the variable. No need to set it to null in this situation because there is only one if-route and you don't need to remember the value afterwards.

Reply:
But according to me, setting a dummy value (or null) for a variable after declaring it is a good practice.

Reply:
That is the case when there is a danger that your variable might be undefined when read.
For example:
```
// let myValue = null;
if (a === true) { myValue = 'a' }
else if (b === true) { myValue = 'b' }
console.log(myValue)
```
In the code you have written, there is no danger of the variable being undefined so there's no reason to set it beforehand to `null`. There are people who like initiating all variables beforehand and that's fine but in this case here `propertyValue` is like a "throwaway variable" that you only need in the `if`-statement just to un-stack the code a bit (you could write the `numberWithCommas(..)` all inside the `.innerText = ..` statement.

Reply:
I thought readability matters a little bit and also, setting some value beforehand should come handy in future coding.